tst/run_all.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Based on tst/testspecial/run_all.sh from GAP
+
+set -ex
+
+retvalue=0
+for testfile in test*.sh; do
+    name="$(cut -d'.' -f1 <<< ${testfile})"
```

Comment:
Huh, what is this doing? Is it a complicated version of `basename PATH EXT` perhaps? Even if it does more than just stripping an extension, then I'd still suggest to use a bash regex instead of calling out to `cut` and using an inline heredoc.

Reply:
Thanks, put `name="$(basename -s .tst ${testfile})"`.